### ISSUE TITLE
Exception raised when resolveWithMib() is invoked on a NotificationType containing objects imported from another MIB

### ISSUE THREAD

**Issue by jeffreymarshall** [etingof/pysnmp#58]
Given the MIBs below, the following code raises an exception:

```
from pysnmp.smi import builder, view, compiler, rfc1902

mibBuilder = builder.MibBuilder()
mibViewController = view.MibViewController(mibBuilder)
compiler.addMibCompiler(mibBuilder, sources=['file:///tmp/mibs'])

mibBuilder.loadModules('SNMPv2-MIB', 'MY-MIB')

oid = rfc1902.ObjectIdentity('1.3.6.1.4.1.99999.2.1')
notificationType = rfc1902.NotificationType(oid).resolveWithMib(mibViewController)
```
Exception
```
Traceback (most recent call last):
  File ".../notification_test.py", line 10, in <module>
    notificationType = rfc1902.NotificationType(oid).resolveWithMib(mibViewController)
  File ".../python2.7/site-packages/pysnmp/smi/rfc1902.py", line 1158, in resolveWithMib
    mibViewController)
  File ".../python2.7/site-packages/pysnmp/smi/rfc1902.py", line 481, in resolveWithMib
    self.__modName, self.__symName
  File ".../python2.7/site-packages/pysnmp/smi/builder.py", line 407, in importSymbols
    'No symbol %s::%s at %s' % (modName, symName, self)
pysnmp.smi.error.SmiError: No symbol MY-MIB::otherMibObj at <pysnmp.smi.builder.MibBuilder object at 0x7eff222f3f50>
```
The root cause seems to be the following line of generated code in MY-MIB.py:
```
myMibNotification = NotificationType((1, 3, 6, 1, 4, 1, 99999, 2, 1)).setObjects(("MY-MIB", "otherMibObj"), ("MY-MIB", "myMibObj"))
```
When manually changed to the following, the exception no longer occurs:
```
myMibNotification = NotificationType((1, 3, 6, 1, 4, 1, 99999, 2, 1)).setObjects(("OTHER-MIB", "otherMibObj"), ("MY-MIB", "myMibObj"))
```

MY-MIB
```
MY-MIB DEFINITIONS ::= BEGIN

IMPORTS
    MODULE-IDENTITY, OBJECT-TYPE, TimeTicks, NOTIFICATION-TYPE, Integer32, enterprises
        FROM SNMPv2-SMI
    otherMibObj
        FROM OTHER-MIB;

myMib MODULE-IDENTITY
    LAST-UPDATED "201705181000Z" -- Thu May 18 10:00:00 PST 2017
    ORGANIZATION "My org"
    CONTACT-INFO "Me"
    DESCRIPTION "Example MIB"
    ::= { enterprises 99999 }

    myMibObj OBJECT-TYPE
        SYNTAX          Integer32 (1..1024)
        MAX-ACCESS      not-accessible
        STATUS          current
        DESCRIPTION "Example object"
        ::= { myMib 1 }

    myMibTraps OBJECT-IDENTITY
        STATUS  current
        DESCRIPTION "The root of all trap OIDs."
        ::= { myMib 2 }


    myMibNotification NOTIFICATION-TYPE
        OBJECTS {
                  otherMibObj,
                  myMibObj
                }
        STATUS  current
        DESCRIPTION
                "Example notification"
        ::= { myMibTraps 1 }

END
```
OTHER-MIB
```
OTHER-MIB DEFINITIONS ::= BEGIN

IMPORTS
    MODULE-IDENTITY, OBJECT-TYPE, Integer32, enterprises
        FROM SNMPv2-SMI;

otherMib MODULE-IDENTITY
    LAST-UPDATED "201705181000Z" -- Thu May 18 10:00:00 PST 2017
    ORGANIZATION "Other org"
    CONTACT-INFO "Other"
    DESCRIPTION "Example MIB"
    ::= { enterprises 99998 }

    otherMibObj OBJECT-TYPE
        SYNTAX          Integer32 (1..1024)
        MAX-ACCESS      not-accessible
        STATUS          current
        DESCRIPTION "Example object"
        ::= { otherMib 1 }


END
```

**Comment by etingof** [comment]
Thank you for raising this issue!

It looks like a bug in pysmi which should be fixed by [eb716b9](https://github.com/etingof/pysmi/commit/eb716b96bc1ee08770e6f9537565c75bb327459c). Could you please try [pysmi master](https://github.com/etingof/pysmi/) and report back in case of any issues?

**Comment by jeffreymarshall** [comment]
Verified the fix works.  Thank you.

**Comment by etingof** [comment]
Just released pysmi 0.1.3 with this fix on [GitHub](https://github.com/etingof/pysmi/releases/tag/v0.1.3) and PyPI.
